charts/sonarqube/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 All changes to this chart will be documented in this file.
 
 ## [10.1.0]
+* Changed default test process to wget, using sonarqube image as default, if you are using a custom image, make sure it contains wget
```

Comment:
I would remove this sentence "if you are using a custom image, make sure it contains wget" from the changelog, as this is more a comment for the user rather that for helm devs.